apps/shooting-clubs/acceptance/features/club-secretary-email.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Club secretary email step');
+
+Before((
+  I,
+  clubSecretaryEmailPage
+) => {
+  I.visitPage(clubSecretaryEmailPage, steps);
+});
+
+Scenario('check there is the correct form elements', (
+  I,
+  clubSecretaryEmailPage
+) => {
+  I.seeElement(clubSecretaryEmailPage['club-secretary-email-id']);
+  I.seeElement(clubSecretaryEmailPage['club-secretary-phone-id']);
```

Comment:
just as an fyi there is a helper method you can use for multiple elements
```
I.seeElements([elements])
```

it also accepts a single element string